src/types/employees.ts

```diff
@@ -16,6 +16,12 @@ export function isChewbaccaEmployeesResponse(
   );
 }
 
+export type Competence =
+  | "Project Management"
+  | "Design"
+  | "Utvikling"
+  | "Administasjon";
```

Comment:
Should utvikling and administrasjon be replaced with english terms?

Reply:
The selected language should ideally match the selected lang: Norwegian for "no," English for "en," and Swedish for "se"

Reply:
Disse er det som kommer fra Chewbacca, kan brukes som nøkler i oversetting